Automattic/wp-calypso
# Domains - Revert "Improve domain actions permissions with new api properties (#61963)"

Related to #62109. This reverts commit c950be1135580f409446850fcbfc73cd383ca637, since it caused some issues with the DNS block on the domain's settings page.

![image](https://user-images.githubusercontent.com/18705930/159344150-9feef5ef-a797-4166-9b34-c924ba99493f.png)

#### Changes proposed in this Pull Request
Reverts commit c950be1135580f409446850fcbfc73cd383ca637.

#### Testing instructions
Go to any domain connection (mapping) settings page and make sure that the DNS block gets loaded properly.
